drafts/use-cases/5.creating-new-event.md

```diff
@@ -0,0 +1,88 @@
+## Creating a new event
+
+### Story
+As an application user
+I want to be able to create new events manually
+So I can have event's I'm aware of added to my calendar.
+
+### Usage
+```javascript
+var event = {
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+};
+var client = new HydraClient();
+var operation = client.get("http://example.com")
+    .getApiDocumentation()
+    .getEntryPoint()
+    .operations
```

Comment:
Looking at JSON-LD payload of the entry point `operations` comes as property of that object https://github.com/HydraCG/Specifications/blob/master/drafts/use-cases/1.entry-point.md#details

I guess accessing it directly assumes that client compacts the response with particular JSON-LD context before returning it from `getEntryPoint()`

I'll talk with @bergos about prototyping reference implementation of client in JS. IMO it would help to use snippets which one can actually execute in something like http://jsbin.com